### PR TITLE
Section skills

### PR DESCRIPTION
**Pull request by Yuliia-Karas**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -34,11 +35,123 @@ button {
...
+
+.section-paragraph {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.5;
+  color: var(--color-paragraph);
+}
+
+.skill-img {
+  background-color: none;
```
Comment (kpovel) [inline css/styles.css]: Mismatched property value

**Review by kpovel — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -17,13 +17,77 @@
...
     <div class="wrapper">
-<header></header>
-<main>
-  <section></section>
-</main>
-<footer></footer>
+      <header></header>
+      <main>
+        <section class="skills">
+          <h2 class="section-title">Skills</h2>
```
Comment (kpovel) [inline index.html:81]: The title should be centered

**Review by kpovel — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -17,13 +17,82 @@
...
-<header></header>
-<main>
-  <section></section>
-</main>
-<footer></footer>
+      <header></header>
+      <main>
+        <section class="container skills">
+          <h2 class="section-title">Skills</h2>
+          <ul class="skills-list">
```
Comment (kpovel) [inline index.html:82]: Check the location of the elements in the layout

Comment (kpovel) [inline index.html:82]: Add rounding to each child of skills-list

**Review by kpovel — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 1 above)
Comment (Yuliia-Karas) [reply css/styles.css]: done

File: index.html (hunk 2 above)
Comment (Yuliia-Karas) [reply index.html:81]: done

File: index.html (hunk 3 above)
Comment (Yuliia-Karas) [reply index.html:82]: done

Comment (Yuliia-Karas) [reply index.html:82]: done

**Comment by Yuliia-Karas** [comment]
fixed bags

File: index.html (hunk 3 above)
Comment (kpovel) [reply index.html:82]: The blocks still have right angles

**Review by kpovel — CHANGES_REQUESTED** [review]
Add responsive layout for the section

**Review by kpovel — APPROVED** [review]

**Pull request merged**